_data/banking.yml

```diff
@@ -780,6 +773,16 @@ websites:
       email_address: santander.email@santander.com.br
       lang: pt
 
+    - name: Santander PL
+      url: https://santander.pl
+      img: santander.png
+      tfa: Yes
+      software: Yes
+      sms: Yes
+      doc: https://santander.pl/firmy/bankowosc-elektroniczna/bzwbk24-mini-firma/metody-autoryzacji/metody-autoryzacji.html
+      exceptions:
+       text: "Software authentication requires bank's mobile app."
```

Comment:
Is the app "iBiznes24 mobile"? If so, change the exception text to "Software authentication only available via iBiznes24 mobile".